### ISSUE TITLE
Create a commit with report in a repo

### ISSUE THREAD

**Issue by pacospace** [thoth-station/slo-reporter#250]
**Is your feature request related to a problem? Please describe.**
As Maintainer/Manager,

I would like to see reports directly into a GitHub Repo.

**Describe the solution you'd like**
- [x] HTML report can be stored using env variable
- [ ] Create/reuse a repo where to store HTML reports that are rendered.

**Describe alternatives you've considered**

**Additional context**
cc @goern

**Comment by goern** [comment]
will we wrap this up in an argo workflow? 1. step to create the report, 2. commit? don't we have that functionality in other workflows?

**Comment by pacospace** [comment]
> will we wrap this up in an argo workflow? 1. step to create the report, 2. commit? don't we have that functionality in other workflows?
Yes in the future wil be an argo CronWorkflow with different steps, as of now, a new method using `source-management` library

**Comment by VannTen** [comment]
Following the discussion in SIG meeting yesterday.

For metrics persistence, would we have the option to retain metrics longer ? (-> and have public dashboard / dashboard snapshots). I know Thanos can store metrics for long-term storage into object storage for example (S3), could that be an option ?

**Comment by goern** [comment]
revisiting the existing dashboards, and esp have the data stored in thanos, seems to be the right thing to do.
